### PR TITLE
OSDOCS-2681: RN for MetalLB 4.10

### PR DESCRIPTION
**Pull request by skrthomas**
Pulling down Mike's updates from https://github.com/openshift/openshift-docs/pull/40351

### CODE REVIEW

**Review by asood-rh — APPROVED** [review]
/lgtm

File: release_notes/ocp-4-10-release-notes.adoc [hunk 1]
```diff
@@ -192,6 +192,21 @@ For more information, see xref:../support/gathering-cluster-data.adoc#support-ne
 
 [id="ocp-4-10-hardware"]
 === Hardware
+=======
```
Comment (ahardin-rh) [inline release_notes/ocp-4-10-release-notes.adoc:195]: Looks like this ======= bit is here by mistake, possibly leftover from a resolved merge conflict.

File: release_notes/ocp-4-10-release-notes.adoc [hunk 2]
```diff
@@ -192,6 +192,21 @@ For more information, see xref:../support/gathering-cluster-data.adoc#support-ne
...
+[id="ocp-4-10-networking-metallb"]
+==== Enhancements to MetalLB load balancing
+
+The following enhancements to MetalLB and the MetalLB Operator are included in this release:
+
+* Support for Border Gateway Protocol (BGP) is added.
+* Support for Bidirectional Forwarding Detection (BFD) in combination with BGP is added.
+* Support for IPv6 and dual-stack networking is added.
+* Support for specifying a node selector on the `speaker` pods is added so you can control which nodes are used for advertising load balancer service IP addresses. This enhancement applies to layer 2 mode and BGP mode.
```
Comment (ahardin-rh) [inline release_notes/ocp-4-10-release-notes.adoc:204]: added. You can now control

**Pull request merged**